ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -160,14 +160,16 @@
             </B.Td>
             <B.Td>
               {{#if (can 'connect target' B.data)}}
-                <Hds::Button
-                  data-test-targets-connect-button
-                  @text={{t 'resources.session.actions.connect'}}
-                  @color='secondary'
-                  @route='scopes.scope.projects.targets.target'
-                  @model={{B.data.id}}
-                  @query={{hash isConnecting=true}}
-                />
+                {{#if B.data.isConnectable}}
```

Comment:
What happens if we don't include this check on the list page and we let the user try and connect? Will it go to the target page and just error?

Reply:
it will just error

Reply:
But by that time we'll have a target loaded right, can we prevent the modal from popping up?

I think we really need to not load all the targets individually, this will cause a massive strain for our current users who have a lot of targets